styles.css

```diff
@@ -19,44 +67,61 @@ body{
     background-color: var(--white-default);
 }
 
-.avatar-container{
-    display: flex;
-    flex-wrap: wrap;
-    column-gap: 2em;
-    text-align: center;
-}
 
-.avatar-small{
-    width: 30px;
-    height: 30px;
-    border-radius: 50%;
-    border: 1px solid var(--white-default);
-    background-color: var(--primary-color);
+.side-nav-bar {
+    margin: 0;
+    padding: 0x;
+    width: 15%;
+    background-color: var(--light-gray);
+    position: fixed;
+    height: 100%;
+    overflow: auto;
+    top: 7.25rem;
   }
-
-.avatar-medium{
-    width: 40px;
-    height: 40px;
-    border-radius: 50%;
-    border: 1px solid var(--white-default);
-    background-color: var(--primary-color);
+  
+  
+.side-nav-bar a {
+    display: block;
+    color: var(--black-default);
+    padding: 1rem;
+    text-decoration: none;
+    font-weight: bold;
   }
 
-.avatar-large{
-    width: 50px;
-    height: 50px;
-    border-radius: 50%;
-    border: 1px solid var(--white-default);
-    background-color: var(--primary-color);
+  .content {
+    margin-left: 15%;
+    padding: 1px 16px;
+    height: 100%;
+    margin-top: 7.25rem;
+    max-width: 100%;
+    overflow: auto;
+    margin-bottom: 7%;
   }
 
+  .footer {
+    background-color: var(--primary-color);
+    text-align: center;
+    padding: 0.2rem;
+    color: white;
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    width: 84.53%;
+}
 
-.avatar-initials{
-    position: relative;
-    top: 30%;
+.footer .footer-heading {
     font-weight: bold;
-    color: var(--white-default);
+    font-size: large;
```

Comment:
It is preferred to use variables with large values instead of mentioning large for better code viewing!!